### ISSUE TITLE
Workspace -"Save" button is always enabled in General Settings, regardless of any changes

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#8531]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to https://staging.new.expensify.com
2. Create a new workspace
3. Tap o new Workspace and change name
4. Tap on Save button

## Expected Result:
"Save" button should be disabled in General Settings when changes are made and saved.

## Actual Result:
"Save" button is always enabled in General Settings, regardless of any changes

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.51.0

**Reproducible in staging?:**  Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** n/a

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/162200527-83e52f72-f97c-4152-b6f7-2d990e93c8a6.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by thesahindia** [comment]
I think we can close this issue as it's not a bug. More info [here](https://github.com/Expensify/App/issues/6717#issuecomment-998034214).

**Comment by techievivek** [comment]
>"Save" button should be disabled in General Settings when changes are made and saved.
I looked at it but the button doesn't respond to clicks when the request is being made and is waiting for the response, so as per the design doc we are in a good state and it shouldn't be a problem. But obviously, we can always prevent the subsequent calls from happening if there are no changes made. Closing this for now because doesn't seem like a big problem. Thanks for highlighting it.
